packages/sdk-metrics/src/aggregator/exponential-histogram/util.ts

```diff
@@ -38,3 +38,17 @@ export function ldexp(frac: number, exp: number): number {
   }
   return frac * Math.pow(2, exp);
 }
+
+export function powTwoRoundedUp(v: number): number {
```

Comment:
This function name to me seems like it would do `Math.pow(inp, 2).ceil()`. I think something like `leastGreaterSquare` or `nextGreaterSquare` might be a bit more clear. This function definitely deserves a tsdoc as well so people don't accidentally call or modify it without knowing what it does.

Reply:
I went with `nextGreaterSquare` and added tsdoc.